src/components/stateful-enroll/data/hooks/useStatefulEnroll.js

```diff
@@ -32,9 +34,16 @@ const useStatefulEnroll = ({
   onSuccess,
   onError,
   onBeginRedeem,
+  courseEnrollmentsByStatus,
 }) => {
   const [transaction, setTransaction] = useState();
-
+  const optimizelyHandler = useOptimizelyEnrollmentClickHandler(
+    contentKey,
+    courseEnrollmentsByStatus,
```

Comment:
[consideration/suggestion] I wonder if we should pull this value out of context rather than prop drilling / passing it as an argument to the hook? Also, similar to a comment above, perhaps we use `userEnrollments` from `CourseContext` here?